### PR TITLE
Include SDK info in Relay exposed metrics

### PR DESCRIPTION
**Pull request by rhcarvalho**

### CODE REVIEW

File: tool/fakerelay/sdkinfo.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	match := t.FindSubmatch(request)
+	if len(match) < 2 {
+		return sdkInfo
+	}
+	s := bytes.SplitN(match[1], []byte("/"), 2)
+	sdkInfo.Name = string(bytes.ReplaceAll(s[0], []byte("-"), []byte(".")))
+	if len(s) > 1 {
+		sdkInfo.Version = string(s[1])
+	}
+	return sdkInfo
```
[the remark this answers is not in the document]
Comment (rhcarvalho) [reply tool/fakerelay/sdkinfo.go:26]: I wouldn't fail the run if it is missing. I guess we'd just note it in the report and if need be change the implementation to deal with other SDKs if we ever need.

File: tool/loadgen/main.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func test(containerName string) TestResult {
 		Stats:   map[string]Stats{"app": stats},
 	}
 	if hasRelay {
-		tr.RelayMetrics = relayMetrics()
+		tr.RelayMetrics = getRelayMetrics()
```
Comment (rhcarvalho) [inline tool/loadgen/main.go:124]: It is not very common to write "getFoo()" in Go, where "foo()" would do. https://golang.org/doc/effective_go#Getters

**Review by AbhiPrasad — APPROVED** [review]

File: template/summary.html.tmpl [hunk 3]
```diff
@@ -336,8 +337,9 @@
       </tr>
       {{ end }}
     </table>
-
-    <hr>
+  
+    <br />
```
Comment (rhcarvalho) [inline template/summary.html.tmpl:341]: In HTML5 this tag is commonly spelled `<br>` (like `<hr>`).

In this case we could just use CSS instead.

Comment (AbhiPrasad) [reply template/summary.html.tmpl]: Yes, probably a better idea to use css.

**Comment by rhcarvalho** [comment]
@AbhiPrasad I backed out everything after the first commit to keep this small and focused. Moving the reporting part to another branch.

**Pull request merged**